src/lib/pubkey/xmss/xmss.h

```diff
@@ -158,6 +158,24 @@ template<typename> class Atomic;
 
 class XMSS_Index_Registry;
 
+/**
+ * Determines how WOTS+ private keys are derived from the XMSS private key
+ */
+enum class WOTS_Derivation_Method
+{
+/// This roughly followed the suggestions in RFC 8391 but is vulnerable
+/// to a multi-target attack. For new private keys, we recommend using
+/// the derivation as suggested in NIST SP.800-208.
+/// Private keys generated with Botan 2.x will need to stay with this mode,
+/// otherwise they won't be able to generate valid signatures any longer.
+Botan2x,
+
+/// Derivation as specified in NIST SP.800-208 to avoid a multi-target attack
+/// on the WOTS+ key derivation suggested in RFC 8391. New private keys
+/// should use this mode.
+NIST_SP800_208,
+};
```

Comment:
Code style: 
```suggestion
   {
   /// This roughly followed the suggestions in RFC 8391 but is vulnerable
   /// to a multi-target attack. For new private keys, we recommend using
   /// the derivation as suggested in NIST SP.800-208.
   /// Private keys generated with Botan 2.x will need to stay with this mode,
   /// otherwise they won't be able to generate valid signatures any longer.
   Botan2x,
   
   /// Derivation as specified in NIST SP.800-208 to avoid a multi-target attack
   /// on the WOTS+ key derivation suggested in RFC 8391. New private keys
   /// should use this mode.
   NIST_SP800_208,
   };
```